index.js

```diff
@@ -3,4 +3,7 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
+var SG = require('strong-globalize');
+SG.SetRootDir(__dirname);
+
```

Comment:
One line would be neater. thought? Please see: https://github.com/strongloop-internal/loopback-knowledge-base/pull/51/files